cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/TaxPage.java

```diff
@@ -0,0 +1,452 @@
+    private WebElement reviewDeductNextButton;
+
+    @FindBy(id = "review-submit-button")
+    private WebElement reviewSubmitButton;
+
+    //Buttons across top of tax page
+
+    @FindBy(id = "file-taxes")
+    private WebElement fileTaxesButton;
+
+    @FindBy(id = "estimate-refund")
+    private WebElement estimateRefundButton;
+
+    @FindBy(id = "document-checklist")
+    private WebElement documentChecklistButton;
+
+    @FindBy(id = "refund-planning")
+    private WebElement refundPlanningButton;
+
+
+
+    //Login Page
+    // private static final String url = "http://localhost:5173/login";
+
+    // public void getMain() {
+    //     try {
+    //         Thread.sleep(1000);
+    //     } catch (InterruptedException e) {
+    //         e.printStackTrace();
+    //     }
+    //     this.driver.get(url);
+    // }
+
+    //login to site
+    // public void login() {
+    //     try {
+    //         Thread.sleep(1000);
+    //     } catch (InterruptedException e) {
+    //         e.printStackTrace();
+    //     }
+    //     emailField.sendKeys("joseph.sam@gmail.com");
+    //     passwordField.sendKeys("password1");
+    //     loginButton.click();
+
+    // }
+    // //Navigate to tax page
+    // public void NavigateToTaxPage() {
+    //     try {
+    //         Thread.sleep(1000);
+    //     } catch (InterruptedException e) {
+    //         e.printStackTrace();
+    //     }
+        
+    //     taxLink.click();
+    // }
+
+    public boolean checkForListOfExistingTaxEstimationRecords() {
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        
+        return driver.findElements(By.xpath("//td[contains(text(),'SINGLE')]")).size() > 0;
+
+    }
+
+    @Override
+    public void clickButton(String name) {
+        WebElement button = getWebElement(name);
+
+        if(button == null) throw new IllegalArgumentException("Button '" + name + "' does not exist.");
+        Wait<WebDriver> wait = new WebDriverWait(driver, Duration.ofSeconds(2));
+    wait.until(d -> button.isDisplayed());
+    try {
+        Thread.sleep(300);
+    } catch (InterruptedException e) {
+        e.printStackTrace();
+    }
+        button.click();
+        
+    }
+
+    public void clickDeleteOrEdit(String button){
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        clickButton(button);
+
+    }
+
+    public void assureTaxRecordAppears(){
+        clickButton("fileTaxesButton");
+        clickButton("estimateRefundButton");
+        clickButton("documentChecklistButton");
+        clickButton("refundPlanningButton");
+        driver.navigate().refresh();
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+
+
+    public boolean checkIfDeleteWorked(){
```

Comment:
Low impact: By name, this method would seem to return true on an item being deleted and false on an item not being deleted; however, the opposite is true.
Suggestion: Add a comment to this method to explain the return and what it checks or give a clearer name.

Reply:
comment added